### PR TITLE
more fixes

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco Please review

File: docs/runbooks/KubevirtVmHighMemoryUsage.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ $ kubectl exec -it <virt-launcher> -c compute -- top
 
 ## Mitigation
 
-Increase the memory limit in the `VirtualMachine` specification as in the following example:
+Increase the memory limit in the `VirtualMachine` specification:
```
Comment (sradco) [inline docs/runbooks/KubevirtVmHighMemoryUsage.md:26]: I think we should mention its an example. No?

Comment (sradco) [reply docs/runbooks/KubevirtVmHighMemoryUsage.md:26]: Or do like for the docs/runbooks/KubeVirtComponentExceedsRequestedMemory.md Where the values are stated like `<memory_limit>` `<memory_request>`

Comment (apinnick) [reply docs/runbooks/KubevirtVmHighMemoryUsage.md:26]: The example in docs/runbooks/KubeVirtComponentExceedsRequestedMemory.md is for upstream only. Would the downstream example look the same?

**Pull request closed without merging**